.github/workflows/ci.yaml

```diff
@@ -416,7 +440,7 @@ jobs:
 
   finalize:
     needs: [metadata, amd64, arm64, publish-containers]
-    if: "!cancelled()"
+    if: '!cancelled()'
```

Comment:
Let's make it consistent:
```
if: ${{ !cancelled() }}
```
WDYT?

Reply:
There are already a lot of consistencies, like https://github.com/NVIDIA/JAX-Toolbox/blob/main/.github/workflows/_ci.yaml#L11 vs https://github.com/NVIDIA/JAX-Toolbox/blob/main/.github/workflows/ci.yaml#L419

I think we should address it in a different PR to not bloat this one. Maybe we should think about some pre-commit checks for linting and formatting?